Tools/dea_tools/app/animations.py

```diff
@@ -162,7 +165,7 @@ def extract_data(self):
             "output_crs": crs,
             "group_by": "solar_day",
             "dask_chunks": {"time": 1, "x": 2048, "y": 2048},
-            "resampling": {"*": "cubic", "oa_fmask": "nearest", "fmask": "nearest"},
+            "resampling": {"*": "bilinear", "oa_fmask": "nearest", "fmask": "nearest"},
```

Comment:
Hey @robbibt, just pointing this out to make sure the change from "cubic" to "bilinear" resampling was deliberate?

Reply:
Good catch - yeah, "bilinear" is fine here (forgot I'd made that change), it's a bit quicker than "cubic" so no need to do that when the output is just for visualisation 😃